### DISCUSSION TITLE
Hello 11.00!

### DISCUSSION THREAD

**Discussion by ultimaweapon** [obhq/obliteration#861] [General]
![Screenshot from 2024-05-05 02-06-00](https://github.com/obhq/obliteration/assets/3048307/554ad5ca-577e-42f3-9f20-a53ede676dac)

**Comment by SoffiMeower** [comment]
AYO-

**Comment by ultimaweapon** [comment]
The next step is to dump an unpatched version. I should be able to finish this by tomorrow. All knowledge I got from reversing 9.0 is really useful when writing a kernel payload.

**Comment by ultimaweapon** [comment]
There are two additional syscalls on 11.00.

**Reply by SoffiMeower** [reply]
Though a lot of 11.00 games easily get backported to 9.00, so the last 2 might not be too important

**Comment by ultimaweapon** [comment]
Seems like there are some memory integrity checking in the PS4 called something like `veri`. If I patch the kernel without https://github.com/TheOfficialFloW/PPPwn/blob/master/stage1/stage1.c#L115 the kernel will panic within minutes after patching. Let's hope this checking is done only on the read-only memory.

**Reply by ultimaweapon** [reply]
Seems like veri check the other things too. I still got a panic even if I restore all kernel patches.

**Comment by ultimaweapon** [comment]
Seems like it is not possible to have execute permission on `kernel_map` with `vm_map_protect`. Never thought getting an unpatched kernel is a complicated task like this.

**Reply by ultimaweapon** [reply]
Seems like I need to call `vm_map_protect` two times with `set_max` set to 1 for the first call and 0 on a second call.

**Reply by ultimaweapon** [reply]
Still crashed. No idea what wrong here. Maybe I should give up on unpatched kernel.

**Reply by ultimaweapon** [reply]
It crashes in `vm_map_protect` somehow.

**Reply by SuchAFuriousDeath** [reply]
Maybe you should. at least for a while.

**Reply by ultimaweapon** [reply]
At least I got one with the patches that does not effect with Obliteration development.

**Comment by SoffiMeower** [comment]
Going from "Hello 11.00!" To "Hell no, 11.00!"
